slicerator/__init__.py

```diff
@@ -540,21 +609,31 @@ def _pipeline_fromclass(cls, retain_doc=False):
     retain_doc : bool
         If True, don't modify `func`'s doc string to say that it has been
         made lazy
+    argc : int or 'all', optional
+        Number of arguments that are relevant for the pipeline. Defaults to 1.
 
     Returns
     -------
     Pipeline
         Lazy function evaluation :py:class:`Pipeline` for `func`.
     """
+    if isinstance(argc, str):
```

Comment:
There could be a Py2/3 issue here. Maybe better: `if argc == 'all':`

Reply:
Fixed.